autoprof/plots/__init__.py

```diff
@@ -1,9 +1,4 @@
-# The Profile module provides a class for reading and parsing profile data
 from .profile import Profile
```

Comment:
There is no `Profile` class, there are a number of methods in the module. Simply `from .profile import *` would be best here. Same for the other imports